paket.dependencies

```diff
@@ -11,4 +11,6 @@ nuget Unquote
 nuget xunit
 nuget xunit.runner.console
 nuget xunit.runner.visualstudio version_in_path: true
-nuget FSharp.Formatting
\ No newline at end of file
+nuget FSharp.Formatting
+nuget Expecto
+nuget System.Threading
```

Comment:
> C:\projects\fscheck\src\FsCheck\Runner.fs(293,34): error FS0039: The field, constructor or member 'Yield' is not defined. [C:\projects\fscheck\src\FsCheck.netcore\FsCheck.netcore.fsproj]

Simply adding System.Threading don't work. Have no idea what to do.

Reply:
Ah, I think I know what's going on there: the netcore project files are not managed via paket (yet, I may change this in the future, but for now paket doesn't yet do exactly what I want with .NET standard). So instead of changing paket.dependencies try adding that package to FsCheck.netcore.fsproj.